connect/src/iframeDisplay.ts

```diff
@@ -81,3 +83,68 @@ export const createIframe = (iframeUrl: string): HTMLIFrameElement => {
   iframe.frameBorder = 0
   return iframe
 }
+
+type DisplayManageMethods = {
+  display: {
+    params: {
+      accountId: string
+    }
+  }
+}
+export class DisplayManageClientRPC {
+  client: RPCClient<DisplayManageMethods>
+
+  constructor(target?: PostMessageTarget) {
+    this.client = createClient<DisplayManageMethods>(NAMESPACE_MANAGE, target)
+  }
+
+  async display(accountId: string): Promise<void> {
+    await this.client.request('display', { accountId })
+  }
+}
+
+export const DisplayManageServerRPC = (manageAppUrl: string, closeCB: () => any): Subscription => {
+  let app: HTMLIFrameElement
+
+  return createServer<DisplayManageMethods>(NAMESPACE_MANAGE, {
+    // todo change name
+    display: async (_event, { accountId }) => {
+      app = createManageIframe(`${manageAppUrl}?accountId=${accountId}`)
+      document.body.appendChild(app)
+
+      await new Promise((resolve) => {
+        app.onload = () => {
+          resolve()
+        }
+      })
+      // @ts-ignore
+      const observer = createMessageObservable(window)
+
+      await new Promise(res => {
```

Comment:
Not an issue but Rx provides APIs to do this, for example: https://github.com/ceramicnetwork/js-transports/blob/main/packages/rpc-transport/src/index.ts#L49

Reply:
ah yes, nice, changed that